src/components/Comment/Comment.vue

```diff
@@ -0,0 +1,211 @@
+<template>
+  <div>
+    <div
+      class="comment mb-3"
+      v-show="comment.is_spoiler === false || isSpoilerClick"
```

Comment:
```suggestion
      v-show="!comment.is_spoiler || isSpoilerClick"
```

Boolean bir değeri boolean bir değer ile eşit olduğunu kontrol etmektense doğrudan kendisi ile işlem yapabilirsin